Junk4Dollars-app/Utils/Time.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+public struct Time {
+    var utc: String
+
+    public init(utc: String) {
+        self.utc = utc
+    }
+
+    public func local() -> String? {
+        let dateFormatter = DateFormatter()
+        let tempLocale = dateFormatter.locale
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+
+
+        guard let date = dateFormatter.date(from: self.utc) else {
+            return nil
+        }
```

Comment:
split this here. Have one method to convert to a Date, and one to convert to a displayable string